webpack.config.js

```diff
@@ -12,6 +12,10 @@ module.exports = {
         test: /\.js?$/,
         exclude: /node_modules/,
         loader: 'babel'
+      },
+      {
+        test: /\.css$/,
+        loader: 'style-loader!css-loader'
```

Comment:
One thing we had before that is now gone is running the css through autoprefixer. You should be able to add that back by using the `postcss-loader` after `css-loader`. Check out `postcss-loader` to see how to use it to get autoprefixing